src/datasources/test-plans/types.ts

```diff
@@ -14,61 +14,166 @@ export enum OutputType {
+        field: ['id'],
+    },
+    [Properties.NAME]: {
+        label: 'Name',
+        projection: ['NAME'],
+        field: ['name'],
+    },
+    [Properties.PROPERTIES]: {
+        label: 'Properties',
+        projection: ['PROPERTIES'],
+        field: ['properties'],
+    },
+    [Properties.STATE]: {
+        label: 'State',
+        projection: ['STATE'],
+        field: ['state'],
+    },
+    [Properties.UPDATED_AT]: {
+        label: 'Updated at',
+        projection: ['UPDATED_AT'],
+        field: ['updatedAt'],
+    },
+    [Properties.UPDATED_BY]: {
+        label: 'Updated by',
+        projection: ['UPDATED_BY'],
+        field: ['updatedBy'],
+    },
+    [Properties.WORKSPACE]: {
+        label: 'Workspace',
+        projection: ['WORKSPACE'],
+        field: ['workspace'],
+    },
+    [Properties.WORK_ORDER]: {
+        label: 'Work order',
+        projection: ['WORK_ORDER_NAME', 'WORK_ORDER_ID'],
+        field: ['workOrderName', 'workOrderId']
+    },
+    [Properties.WORK_ORDER_ID]: {
+        label: 'Work order ID',
+        projection: ['WORK_ORDER_ID'],
+        field: ['workOrderId'],
+    },
+    [Properties.PRODUCT]: {
+        label: 'Product (Part number)',
+        projection: ['PART_NUMBER'],
+        field: ['partNumber'],
+    },
+    [Properties.PLANNED_START_DATE_TIME]: {
+        label: 'Planned start date/time',
+        projection: ['PLANNED_START_DATE_TIME'],
+        field: ['plannedStartDateTime'],
+    },
+    [Properties.ESTIMATED_END_DATE_TIME]: {
+        label: 'Estimated end date/time',
+        projection: ['ESTIMATED_END_DATE_TIME'],
+        field: ['estimatedEndDateTime'],
+    },
+    [Properties.ESTIMATED_DURATION_IN_SECONDS]: {
+        label: 'Estimated duration',
+        projection: ['ESTIMATED_DURATION_IN_SECONDS'],
+        field: ['estimatedDurationInSeconds'],
+    },
+    [Properties.SYSTEM_NAME]: {
+        label: 'System name',
+        projection: ['SYSTEM_ID'],
+        field: ['systemId'],
+    },
+    [Properties.SYSTEM_ID]: {
+        label: 'System ID',
+        projection: ['SYSTEM_ID'],
+        field: ['systemId'],
+    },
+    [Properties.TEMPLATE]: {
+        label: 'Test plan template',
+        projection: ['TEMPLATE_ID'],
+        field: ['templateId'],
+    },
+    [Properties.TEMPLATE_ID]: {
+        label: 'Template ID',
+        projection: ['TEMPLATE_ID'],
+        field: ['templateId'],
+    },
+    [Properties.TEST_PROGRAM]: {
+        label: 'Test program name',
+        projection: ['TEST_PROGRAM'],
+        field: ['testProgram'],
+    },
+    [Properties.SUBSTATE]: {
+        label: 'Substate',
+        projection: ['SUBSTATE'],
+        field: ['substate'],
+    },
+    [Properties.FIXTURE_NAMES]: {
+        label: 'Fixture names',
+        projection: ['FIXTURE_IDS'],
+        field: ['fixtureIds'],
+    },
+    [Properties.DUT_ID]: {
+        label: 'DUT',
+        projection: ['DUT_ID'],
+        field: ['dutId'],
+    },
 };
```

Comment:
Should mark exported const objects `as cont`

```suggestion
} as const;
```

Reply:
Applied suggestion.